flatland_plugins/include/flatland_plugins/laser.h

```diff
@@ -79,6 +79,9 @@ class Laser : public ModelPlugin, public b2RayCastCallback {
   std::string frame_id_;  ///< laser frame id name
   bool broadcast_tf_;     ///< whether to broadcast laser origin w.r.t body
   uint16_t layers_bits_;  ///< for setting the layers where laser will function
+  uint16_t reflectance_layers_bits_;  ///< for setting reflectance layers. if
+                                      ///laser hits those layers, intensity will
+                                      ///be high
```

Comment:
these two lines are the ones causing the travis-ci to fail. It wants them to not be intented. Maybe move to a block quote before the definition?

Check with 

> git ls-files | grep -E "\.[ch](pp)?$" | grep -v "thirdparty/" |  xargs clang-format-3.8 --style=file -i && git diff --exit-code